### PR TITLE
Message embed

### PR DESCRIPTION
**Pull request by devsnek**

### CODE REVIEW

File: src/structures/MessageEmbed.js [hunk 1]
```diff
@@ -121,265 +179,169 @@ class MessageEmbed {
...
-
-    /**
-     * If this field is displayed inline
-     * @type {boolean}
-     */
-    this.inline = data.inline;
+  /**
+   * Create an object that can be sent in a message to discord
+   * @returns {Object}
+   */
```
Comment (iCrawl) [inline src/structures/MessageEmbed.js]: Private?

Comment (GusCaplan) [inline src/structures/MessageEmbed.js]: probably

Comment (iCrawl) [inline src/structures/MessageEmbed.js]: Then make it so.

**Comment by iCrawl** [comment]
This still needs to be finished. @GusCaplan

**Comment by GusCaplan** [comment]
@Drahcirius merge conflicts plz fix

**Comment by iCrawl** [comment]
#1529

**Pull request closed without merging**